### DISCUSSION TITLE
next/image Internal server error in Production mode

### DISCUSSION THREAD

**Discussion by brkdmn35** [vercel/next.js#33046] [Help]
### Run `next info` (available from version 12.0.8 and up)

```js
{
  name: 'next-boilerplate',
  'dist-tags': {
    latest: '0.0.0'
  },
  versions: [
    '0.0.0'
  ],
  time: {
    created: '2020-07-08T09:16:42.255Z',
    '0.0.0': '2020-07-08T09:16:42.382Z',
    modified: '2020-07-08T09:16:44.623Z'
  },
  maintainers: [
    {
      name: 'bunlong',
      email: 'bunlong.van@gmail.com'
    }
  ],
  license: 'ISC',
  version: '0.0.0',
  main: 'index.js',
  scripts: {
    test: 'echo "Error: no test specified" && exit 1'
  },
  dist: {
    integrity: 'sha512-q4hzYgVGLyGKX7OdwnG9LlAcmIaGF3fRWCWpkhgQbg0CtoPWq6O+gsDI2DcOD2mHRA+GE2Fbk5yWWD4pcXdx8Q==',
    shasum: 'a97fd29625354bc06806bb27914014643efdb87d',
    tarball: 'https://registry.npmjs.org/next-boilerplate/-/next-boilerplate-0.0.0.tgz',
    fileCount: 1,
    unpackedSize: 212,
    'npm-signature': '-----BEGIN PGP SIGNATURE-----\r\n' +
      'Version: OpenPGP.js v3.0.4\r\n' +
      'Comment: https://openpgpjs.org\r\n' +
      '\r\n' +
      'wsFcBAEBCAAQBQJfBY76CRA9TVsSAnZWagAAOZkP/09LVyR9YWLvkAp2YS3W\n' +
      'eMNAFjjozpV/cXnvFKjIJPGka4WwNa9OoSa3WJJtU+xCJ4zQP4b8n4xx9AHm\n' +
      '9d5Et78ynGtuH5bMDa4DB0pDUTdGCMxKuCHCtG0g+QoAJDqt7QO3E1Un5BOj\n' +
      'tSuulaxPwh24emCX5xzbMJxI/QvaFG5D4rsD/PZVFuh8NpxF05p80n0X6W24\n' +
      'yampXy5VGD6EitUUIvCa1k2k9K2Ppgq7xSWtV4xycw07RLxMak9zjJusc+4t\n' +
      'wQH496S251ii7jXygDaGXunQlVsbxoxH6u0YzFkMeFcbggJZcIyGZdQNbcm0\n' +
      '7Xsd1Q0P7xzRZ8m6bPgYAuSl06p7dJ+ey3ucfQ8O+z/VYYM6PiG63RTPhRn1\n' +
      'QZM/934QHDZ3wqNrUL/EyARWOssslJBX190K5STgD8rgqSv05cmuUq3TWuSM\n' +
      '/mBywZbcuKaxRkH18bGDtzhex7cH27pbR3aq3Prtnry9ss4e4vSrnR2vdZyi\n' +
      '/krn5iHyfFlBcpQB/jJej87RfZ61PE8nP8QLYX0eToluUFUxyUkwbKy1WjrR\n' +
      'vdVM38mX0/8Q3a8wKbc/rJNtUNn2bKoraC36MMKi4SXwAS09aiv94EZRbn4d\n' +
      'Tb8m7fTnp5dljsTh/swYn7HanjUBXSNF25vvgaByGHW7QKgNUHA/ZNq/98I6\n' +
      'nW5J\r\n' +
      '=u4tc\r\n' +
      '-----END PGP SIGNATURE-----\r\n'
  },
  directories: {}
}
```

Done in 1.14s.

### What version of Next.js are you using?

12.0.0

### What version of Node.js are you using?

14.17.1

### What browser are you using?

Chrome

### What operating system are you using?

windows

### How are you deploying your application?

AWS on Docker

### Describe the Bug

Hello,

I just wanter to use next/image for optimised images. Currently we are using cdn provider.

I implemented the domains to next.config.js. In development mode everything is fine. But when I get build the project and run with Production mode. Images fail with 500 status (Internal Server Error)

### Expected Behavior

If I give the src without ssl works fine http://cdn.hesapkurdu.com/images/bank/m-14.png but with ssl https://cdn.hesapkurdu.com/images/bank/m-14.png it gives error. So, is there any settings I am missing or is this bug. I am not sure.

- It gives this error on console

`FetchError: request to https://cdn.hesapkurdu.com/images/bank/m-15.png failed, reason: unable to verify the first certificate
    at ClientRequest.`

- I also added next.config.js to build path in Docker build.

### To Reproduce

```js
module.exports = {
    images: {
        domains: [ //Image domains should be defined here
            'cdn-staging-static.hesapkurdu.com',
            'cdn-staging.hesapkurdu.com',
            'cdn-static.hesapkurdu.com',
            'qm1n93v26l.mncdn.com',
            'cdn.hesapkurdu.com',
            'localhost',
            'localhost:3000'
        ],
    },
    poweredByHeader: false,
    swcMinify: true,
    
}



```

**Comment by masdimya** [comment]
have you solve this issue ? i have similar issue with next/image 500 status (Internal Server Error) in production

**Comment by yattdev** [comment]
Hi here ! i facing similar issue with next/image 500 status (Internal Server Error) in production, have you solve this ?
**Version of Next.js**: 10.0.9

**Reply by brkdmn35** [reply]
I couldn't find any solution yet. Will be work on after solved other problems

**Reply by yattdev** [reply]
Thank to reply. Have you an image optimise solution/alternative, That can help me ?

**Comment by ianzablovschii** [comment]
I did have the same issue. In my case the problem was the wrong permissions on the .next/cache folder. NextJS did not have the permission to save cached images in that folder.

**Reply by joaotextor** [reply]
Hello. And how did you solve this problem?

**Reply by ianzablovschii** [reply]
@joaotextor Add more permissions to the .next/cache folder. As a test, try `chown -R nextjs:nodejs ./.next`

**Comment by Carlosdp7** [comment]
Cloud anyone solve the problem? I have the same issue.

**Comment by 0sama** [comment]
I fixed it by passing the `unoptimized` option to the `<Image />` component.
I didn't need it since I was already doing image optimization on the CDN itself.
